lib/elasticsearch.ex

```diff
@@ -507,7 +507,7 @@ defmodule Elasticsearch do
     end)
   end
 
-  defp format({:ok, %{status_code: code, body: body}})
+  defp format({:ok, %{status: code, body: body}})
        when code >= 200 and code < 300 do
```

Comment:
btw, you can also do slightly more compact:

```suggestion
       when code in 200..299 do
```